### ISSUE TITLE
Product "boxes" are not aligning correctly.

### ISSUE THREAD

**Issue by chriskacerguis** [projectjellyfish/api#451]
Product "boxes" are not aligning correctly on 

/project/:id 
OR
/marketplace/list
OR
/marketplace/show/:id

![screen shot 2015-03-30 at 3 31 38 pm](https://cloud.githubusercontent.com/assets/2414647/6906099/30a20848-d6f2-11e4-8dc1-4906d0ee19f0.png)

**Comment by sonejah21** [comment]
Fixed the product icon placement and project status icon placement using "-webkit-" before the css class "transform". Pushed these changes up.

After fixing the original problem (above) I noticed that the content of the product boxes in the marketplace while in their "hover" state is off. 

@calebthompson and @chriskacerguis - could this issue possibly be from the merge? any ideas why font awesome doesn't load in safari? I couldn't snap a screenshot of the hover state, but I captured the error messages upon reloading the page a few times. See below.

![uploads/e7756364-a867-474a-b582-97178ea1e396/Screen Shot 2015-03-31 at 2.29.20 PM.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2Fe7756364-a867-474a-b582-97178ea1e396%2FScreen+Shot+2015-03-31+at+2.29.20+PM.png)
